sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/ActivityTagsProcessor.cs

```diff
@@ -45,18 +44,8 @@ internal struct ActivityTagsProcessor
             SemanticConventions.AttributeNetPeerName,
             SemanticConventions.AttributeNetPeerIp,
             SemanticConventions.AttributeNetPeerPort,
-            SemanticConventions.AttributeNetTransport,
-            SemanticConventions.AttributeNetHostIp,
             SemanticConventions.AttributeNetHostPort,
             SemanticConventions.AttributeNetHostName,
-            SemanticConventions.AttributeComponent,
-            "otel.status_code",
```

Comment:
Why this was removed?

Reply:
This was intentional, this was a legacy attribute that provided status support in OpenTelemetry.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/24250/files

Reply:
The PR you shared has zero comments and doesn't explain why this change was made.

If this is a legacy attribute, I would expect no Instrumentation Libraries to emit this. I expect it would be safe to remove it here. I'll ping you offline, there must be some detail I'm not understanding.

Reply:
spoke offline. This is considered legacy because it was replaced by the top level property `Activity.Status`.
We're leaving this here so it gets filtered out and omitted from telemetry.